composer.json

```diff
@@ -15,6 +15,27 @@
 		"guzzlehttp/promises": "~1.0",
 		"psr/log": "~1.0"
 	},
+	"require-dev": {
+		"phpunit/phpunit": "~4.8.0|~5.3.0",
+		"jakub-onderka/php-parallel-lint": "0.9.2"
+	},
+	"suggest": {
+		"etsy/phan": "Allows running static analysis on the package (requires PHP 7+)"
```

Comment:
Suggest is meant for packages using the one where you put it if I'm not mistaken. This one does not seem to make a lot of sense like that